README.md

```diff
@@ -6,6 +6,22 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/operator-framework/operator-lifecycle-manager)](https://goreportcard.com/report/github.com/operator-framework/operator-lifecycle-manager)
 [![Slack Channel](https://img.shields.io/badge/chat-4A154B?logo=slack&logoColor=white "Slack Channel")](https://kubernetes.slack.com/archives/C0181L6JYQ2)
 
+---
+<span style="background-color:yellow">
+[!WARNING]  
+This repository is associated with **Operator Lifecycle Manager** version 0 (v0) which is in maintenance mode. The community is excited to be working on a new design (v1) which intends to overcome a number of limitations of the v0 architecture, the primary repository for which is [operator controller](https://www.github.com/operator-framework/operator-controller).  
+</span>
```

Comment:
To use GitHub's native markdown styling for this it should be:
```suggestion
> [!WARNING]  
> This repository is associated with **Operator Lifecycle Manager** version 0 (v0) which is in maintenance mode. The community is excited to be working on a new design (v1) which intends to overcome a number of limitations of the v0 architecture, the primary repository for which is [operator controller](https://www.github.com/operator-framework/operator-controller).  
```
which will display as:

> [!WARNING]
> Be careful!

For reference: https://github.com/orgs/community/discussions/16925

Reply:
I was experimenting with formats, but at the very least it looks like my experiment doesn't work in the 'view rich diffs' panel of an in-review PR. :(